resources/prometheus/federation-config.yaml

```diff
@@ -1,4 +1,171 @@
 match[]:
-  # Federate all platform metrics except ACS metrics. We want all base metrics for the Kubernetes mixin.
-  - '{__name__=~".*", job!="central"}'
-  - '{__name__=~".*", job="central", endpoint!="monitoring-tls"}'
+  - :node_memory_MemAvailable_bytes:sum{job!~"central|scanner"}
```

Comment:
I guess we need to keep in mind that once scanner-v4 components land in ACS-CS, we use label `job=scanner` for them 🤔 
cc @mclasmeier

Reply:
@porridge, it is not relevant to use `scanner` as the job name as long we are not defining that job for Openshift Prometheus. Currently, there are no scanner scrape jobs in OSD Prometheus.